src/app/organization/tellers/allocate-cash/allocate-cash.component.ts

```diff
@@ -0,0 +1,88 @@
+/** Angular Imports. */
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DatePipe } from '@angular/common';
+
+/** Custom Services. */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/**
+ * Allocate Cash component.
+ */
+@Component({
+  selector: 'mifosx-allocate-cash',
+  templateUrl: './allocate-cash.component.html',
+  styleUrls: ['./allocate-cash.component.scss']
+})
+export class AllocateCashComponent implements OnInit {
+
+  /** Cashier data. */
+  cashierData: any;
+  /** Cashier Form. */
+  allocateCashForm: FormGroup;
+  /** Minimum Date allowed. */
+  minDate = new Date(2000, 0, 1);
+  /** Maximum Date allowed. */
+  maxDate = new Date();
```

Comment:
shift above cashierdata and form variables

Reply:
Please check

Reply:
place min and max date as first 2 variables

Reply:
Resolved